contracts/contracts/coordination/Coordinator.sol

```diff
@@ -359,13 +391,13 @@ contract Coordinator is AccessControlDefaultAdminRules {
     }
 
     function getParticipantFromProvider(
-        uint256 ritualID,
+        uint32 ritualID,
```

Comment:
Should we be consistent with our camel-case (other spots as well - L400 and L412)?
```suggestion
        uint32 ritualId,
```

Reply:
✔️